src/utils.js

```diff
@@ -115,10 +140,9 @@ const getDefaultValue = (options: GetDefaultValueOptions) => {
       const fieldPath = has(field, 'questionId') ? mergePaths(pathPrefix, field.questionId) : pathPrefix;
       if (has(field, 'questionId') && !has(defaultValues, fieldPath)) {
         // might need isNilOrEmpty(get(defaultValues, fieldPath)) in the future
-        if (has(field, 'defaultValue')) {
-          set(defaultValues, fieldPath, field.defaultValue);
-        } else if (has(fieldOptions, '_genDefaultValue')) {
-          set(defaultValues, fieldPath, fieldOptions._genDefaultValue);
+        const {hasDefaultValue, defaultValue} = getDefaultValueHelper({field, fieldOptions});
```

Comment:
Are you anticipating this getDefaultValueHelper to be used elsewhere? Right now you refactored for just this one location.

Reply:
It's also used in `GenCondClearField` to get the `defaultValue`